src/main/java/org/broadinstitute/consent/http/db/VoteDAO.java

```diff
@@ -45,7 +44,7 @@ Vote findVoteByElectionIdAndUserId(@Bind("electionId") Integer electionId,
   Integer checkVoteById(@Bind("referenceId") String referenceId,
       @Bind("voteId") Integer voteId);
 
-  @SqlUpdate("insert into vote (user_id, electionId, type, reminderSent) values (:userId, :electionId, :type, false)")
+  @SqlUpdate("INSERT INTO vote (user_id, electionid, type, remindersent, createdate) VALUES (:userId, :electionId, :type, false, current_timestamp)")
```

Comment:
Note that this was not setting the create date which left some code paths creating votes with null values.

Reply:
We expanded `VoteDAOTest. testFindVoteById()` to check for the create date on line 53.